assets/certs/Makefile

```diff
@@ -1,5 +1,28 @@
-all:
-	for f in etcd1 etcd2 etcd3 proxy1; do \
-		openssl req -new -key $$f-key.pem -out $$f.csr -config config; \
-		openssl x509 -req -days 1000 -CA ca.pem -CAkey ca-key.pem -CAcreateserial -in $$f.csr -out $$f.pem -extfile config -extensions 'v3_req'; \
-	done
+CERTS = etcd1.crt etcd2.crt etcd3.crt proxy1.crt
+
+.PHONY: all
+all: $(CERTS)
+
+.PHONY: ca
+ca: ca.crt
+
+%.key:
+	openssl genrsa -out $@ 2048
+
+ca.csr: ca.key ca.config
+	openssl req -new -key $< -out $@ -config ca.config
+
+ca.crt: ca.csr ca.key
+	cat ca.csr | openssl x509 -req -days 1095 -sha256 -keyform pem -signkey ca.key -set_serial 0x2EC9165BAC1CFD93B558790F8EB291D99E359072 -out ca.crt
```

Comment:
`0x2EC9165BAC1CFD93B558790F8EB291D99E359072` was pulled off the existing CA certificate.